dll/win32/shlwapi/ordinal.c

```diff
@@ -3293,8 +3293,8 @@ DWORD WINAPI SHGetIniStringW(LPCWSTR appName, LPCWSTR keyName, LPWSTR out,
 {
 #ifdef __REACTOS__
     WCHAR szSection[MAX_PATH + 2];
-    WCHAR szWideBuff[MAX_PATH];
-    CHAR szUtf7Buff[MAX_PATH];
+    CHAR szUtf7Buff[4 * MAX_PATH]; /* UTF-7 needs 4 times length buffer */
+    WCHAR szWideBuff[4 * MAX_PATH]; /* UTF-7 in UTF-16 */
```

Comment:
If those buffers could be made dynamic, it would be better so that less stack space gets used.
(Same comment for the other function and for the new `CIniPropertyBag::Read`.

Reply:
Fixed.